summary.Rmd

```diff
@@ -0,0 +1,40 @@
+Summary
+=======
+
+1. Use available tools
+    a. Use ready-made images (e.g. from Docker Hub)
+    b. Dockerfile generators (e.g. repo2docker)
+    c. Project templates (e.g. [cookie cutter](https://github.com/cookiecutter/cookiecutter))
+    d. Linters
+2. Use versioned images
```

Comment:
a. Use version-tagged base images
b. Publish your own images with version tags

Though "b" is not in the rule - yet?

Reply:
Done! I'll leave the main text until this is settled but I've added some tags to the summary points to indicate changes to be made. Indeed, "b" is not explicit in the main text yet but it only needs a small tweak to add it so I have added the [revise] tag in the summary.